Improve landing page (#18)
ambanum/disinformation-encyclopedia: _pages/index.md

permalink: /
nav_exclude: true
---
<div>
	<section class="hero is-medium is-bold">
		<div class="hero-head">
			<header class="navbar">
				<div class="container">

> 					<div class="navbar-brand">

Reviewer: > So many `<div>`…  😞

Author: Structure required by Bulma

> 						<img class="logo" src="https://avatars3.githubusercontent.com/u/7874148?s=400&v=4" alt="Logo">

Reviewer: Either empty `alt` if the image is decorative or a meaningful alt.
```suggestion
						<img class="logo" src="https://avatars3.githubusercontent.com/u/7874148?s=400&v=4" alt="">
```

					</div>
					<div class="navbar-menu">
						<div class="navbar-end">

> 							<a href="/collaborate" class="navbar-item" data-proofer-ignore>

Reviewer: Add a comment to explain this `data-proofer-ignore` attribute.

								Collaboration
							</a>

> 							<a href="/encyclopedia" class="navbar-item">

Reviewer: Shouldn't this be prefixed with `{{ baseurl }}` in order to allow deployments that don't make the assumption of being hosted at root?

Author: I don't think it's necessary for now

								Encyclopedia
							</a>
							<a href="/encyclopedia" class="navbar-item">
								About

> 							</a>

Reviewer: Let's remove this link until we have a proper `about` page, right now this is misleading.

						</div>
					</div>
				</div>
			</header>
		</div>
		<div class="hero-body">
			<div class="container has-text-centered">
				<h1 class="title is-spaced">
					Fight against disinformation
				</h1>
				<h2 class="subtitle">
					We aim at reinforcing the democratic antibodies that are present in our societies.<br/>
					We are building the place where those who work towards coherence and authenticity gather.
				</h2>
			</div>
		</div>
	</section>
	<section class="section our-work">
		<div class="container">
			<h1 class="our-work-title title has-text-centered">
				What we're currently working on…
			</h1>
			<div class="columns">
				<div class="column is-5 is-offset-1">
					<div class="work-item">
						<h2 class="title is-spaced">

> 							<span class="thin">Collaboration</span>chat

Reviewer: Add a space between words. #a11y #seo
```suggestion
							<span class="thin">Collaboration</span> chat
```

						</h2>
						<p>

> 							Our main focus at the moment is a chat on steroids: a real-time communication channel where all detection, analysis and reaction can happen in a shared fashion with mutualized tools. You’ll find more about the chatbots we have built <a href="/encyclopedia/chatbots/">here</a>. If you’re reading this we’d probably like to have you on board. <a href="mailto:matti.schneider@diplomatie.gouv.fr">Send us an email!</a>

Reviewer: ```suggestion
							Our main focus at the moment is a chat on steroids: a real-time communication channel where all detection, analysis and reaction can happen in a shared fashion with mutualized tools. Learn more about the <a href="/encyclopedia/chatbots/">chatbots we have built</a>. If you’re reading this we’d probably like to have you on board. <a href="mailto:matti.schneider@diplomatie.gouv.fr">Send us an email!</a>
```

> 							Our main focus at the moment is a chat on steroids: a real-time communication channel where all detection, analysis and reaction can happen in a shared fashion with mutualized tools. You’ll find more about the chatbots we have built <a href="/encyclopedia/chatbots/">here</a>. If you’re reading this we’d probably like to have you on board. <a href="mailto:matti.schneider@diplomatie.gouv.fr">Send us an email!</a>

Reviewer: Shouldn't `baseurl` be somewhere?

						</p>
						<a href="/collaborate" class="button is-white is-medium" data-proofer-ignore>
							Collaborate
						</a>
					</div>
				</div>
				<div class="column is-5">
					<div class="work-item">
						<h2 class="title is-spaced">
							<span class="thin">Dis</span>Encyclopedia
						</h2>
						<p>
							We are documenting their practices and equipping them with the best tools available, with the intention that they will share their results with each other while keeping their independence. 
							We provide them with opportunities to improve and grow, and identify the most effective ways to decrease the impact of disinformation.
						</p>
						<a href="/encyclopedia" class="button is-white is-medium">
							Learn
						</a>
					</div>
				</div>
			</div>
		</div>
	</section>

> 	<footer class="footer">

Reviewer: Why duplicate the information through a class when the tag name is semantic enough? 🤔 
```suggestion
	<footer>
```

Author: Required by Bulma CSS framework

		<div class="container has-text-centered">
			<div class="columns">
				<div class="column is-full">
					<a href="/encyclopedia">About</a>

> 					&nbsp;|&nbsp;

Reviewer: > This would be more appropriate as `border-right`.
